corehq/apps/users/bulkupload.py

```diff
@@ -277,7 +277,7 @@ def create_or_update_groups(domain, group_specs, log):
     group_names = set()
     for row in group_specs:
         group_id = row.get('id')
-        group_name = row.get('name')
+        group_name = unicode(row.get('name') or '')
```

Comment:
Are the values you get here always ASCII bytes? If not, this could blow up. Same below.